lldb/source/Interpreter/CommandReturnObject.cpp

```diff
@@ -112,8 +113,46 @@ void CommandReturnObject::SetError(Status error) {
 }
 
 void CommandReturnObject::SetError(llvm::Error error) {
-  if (error)
+  // Retrieve any diagnostics.
+  error = llvm::handleErrors(std::move(error), [&](ExpressionErrorBase &error) {
```

Comment:
Why is this data structure called ExpressionErrorBase?  That's a clearly inappropriate name for errors in a generic CommandObject, and seeing it here is confusing.

Reply:
I renamed it to DiagnosticError and moved it to Utility.